subsystems/viamagent/viamagent.go

```diff
@@ -24,8 +24,8 @@ func init() {
 
 const (
 	subsysName      = "viam-agent"
-	serviceFileDir  = "/etc/systemd/system"
-	serviceFilePath = "/etc/systemd/system/viam-agent.service"
+	serviceFileDir  = "/usr/local/lib/systemd/system"
```

Comment:
what would be the downside of not doing this? is this mostly a best practice cleanup?

Reply:
So Pete found that, when he tried to remove the "After=" line from the service file with `systemctl edit --full viam-agent` it REPLACES the file in /etc/ as that's meant to be the user/admin override location. The downside is that, on next update, we then try to "repair" that file with a new copy, losing his customizations.

The short version is /etc is for user/owner config, whereas /usr/lib/ is for vendoer/OS installed services (stuff controlled by the OS's package manager), and /usr/local/lib/ is where stuff in between (like us) goes. And the other locations all can be ultimate overridden by the user in /etc.

Reply:
Ohh that's a good observation